### ISSUE TITLE
More guidance on exercise in nb7

### ISSUE THREAD

**Issue by eor314** [NOC-OI/computer-vision-workshop#6]
The exercise material in notebook 7 current reads: 

``` python
# load the network weights
spc_model = torch.load('spc_resnet.pt')

# set it to evaluation mode
spc_model.eval()
```

This is too skeletal and participants struggled when they got to here. Providing more guidance about the necessary steps in needed:

- Load Model Weights
- Load ResNet18 Architecture
- Apply a custom fully connected layer to your ResNet18 Architecture
- Apply the weights and evaluate on the validation set.

This could be either as comments in the code block, new code blocks with explanatory text, etc. Ideally, it **will not** be function calls with blanks to fill in. The exercise should be a bit of a step up from previous ones.
